README.md

```diff
@@ -6,6 +6,7 @@
 
 <!--
     Copyright 2025 MNX Cloud, Inc.
+    Copyright 2025 Edgecast Cloud LLC.
```

Comment:
Just for a check, we do not replace MNX line?

Reply:
@danmcd and I discussed on Slack and we concluded that adding rather than replacing was appropriate.
[see also](https://github.com/TritonDataCenter/triton-moirai/pull/5#discussion_r2229244497).